templates/server/partial/_server-footer.html

```diff
@@ -0,0 +1,64 @@
+<div class="p-section">
```

Comment:
Design for this looks slightly different (heading / "download" in the same row), could you take a look?

Reply:
Thanks, fixed!